### ISSUE TITLE
Updates overwrite custom themes

### ISSUE THREAD

**Issue by davidjorg** [platformio/platformio-atom-ide-terminal#450]
Every time a new platformio-ide-terminal update is installed, manually created themes stored in .../.atom/packages/platformio-ide-terminal/styles/themes and registered in themes.less and platformio-ide-terminal.coffee are removed and have to be reinstalled by hand. Is it possible to fix this?

**Comment by the-j0k3r** [comment]
Not really, best way is to automate re-adding them via a script or something.
